### ISSUE TITLE
[Enhancement]:

### ISSUE THREAD

**Issue by raquel-emberly** [woothemes/woocommerce#60719]
### Describe the solution you'd like

On the cart page when products have been added to the cart

Product table > Reduce quantity button
This element should be moved before the quantity input in the HTML so that screen reader users will encounter this element in an unsurprising order and focus will follow a logical sequence for keyboard users.

When resolving this issue, do not use tabindex attributes to change the default focus order. Additionally, do not use JavaScript to actively manage the keyboard focus based on keypresses.

<img width="1884" height="529" alt="Image" src="https://github.com/user-attachments/assets/8e7e891e-1b30-4bc5-9ef8-a6d7ea87d002" />

Affected users
Motor / Dexterity, Visual

Guidelines
[1.3.2 (WCAG 2.0 A)Present content in a meaningful order](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence), [2.4.3 (WCAG 2.0 A)Content receives focus in a logical and predictable order.](https://www.w3.org/WAI/WCAG22/Understanding/focus-order)

**Comment by nerrad** [comment]
This is a duplicate of #60719
